static/webpack.config.js

```diff
@@ -32,10 +32,10 @@ const config = {
     ],
     dev: [__dirname + "/js/dev.ts", __dirname + "/css/dev.scss"],
     timeline: [
-      __dirname + "/js/tools/timeline.ts",
+      __dirname + "/js/tools/timeline/timeline.ts",
       __dirname + "/css/timeline.scss",
     ],
-    timeline_bulk_download: [__dirname + "/js/tools/timeline_bulk_download.ts"],
+    timeline_bulk_download: [__dirname + "/js/tools/timeline/bulk_download.ts"],
```

Comment:
dont forget to also update the corresponding html file

Reply:
Just tested the js name is based on the entry key "timeline_bulk_download" so this should be good.